std/node/process.ts

```diff
@@ -118,53 +285,34 @@ export const process = {
         notImplemented();
       },
     };
-  },
-
-  /** https://nodejs.org/api/process.html#process_process_events */
-  // on is not exported by node, it is only available within process:
-  // node --input-type=module -e "import { on } from 'process'; console.log(on)"
-  // deno-lint-ignore ban-types
-  on(_event: string, _callback: Function): void {
-    // TODO(rsp): to be implemented
-    notImplemented();
-  },
-
-  /** https://nodejs.org/api/process.html#process_process_argv */
-  get argv(): string[] {
-    // Getter delegates --allow-env and --allow-read until request
-    // Getter also allows the export Proxy instance to function as intended
-    return [Deno.execPath(), ...Deno.args];
-  },
-
-  /** https://nodejs.org/api/process.html#process_process_env */
-  get env(): { [index: string]: string } {
-    // Getter delegates --allow-env and --allow-read until request
-    // Getter also allows the export Proxy instance to function as intended
-    return Deno.env.toObject();
-  },
-  nextTick,
-};
+  }
 
-/**
- * https://nodejs.org/api/process.html#process_process_argv
- * @example `import { argv } from './std/node/process.ts'; console.log(argv)`
- */
-// Proxy delegates --allow-env and --allow-read to request time, even for exports
-export const argv = new Proxy(process.argv, {});
+  /** https://nodejs.org/api/process.html#process_process_version */
+  version = version;
 
-/**
- * https://nodejs.org/api/process.html#process_process_env
- * @example `import { env } from './std/node/process.ts'; console.log(env)`
- */
-// Proxy delegates --allow-env and --allow-read to request time, even for exports
-export const env = new Proxy(process.env, {});
+  /** https://nodejs.org/api/process.html#process_process_versions */
+  versions = versions;
+}
 
-// import process from './std/node/process.ts'
-export default process;
+/** https://nodejs.org/api/process.html#process_process */
+const process = new Process();
 
 Object.defineProperty(process, Symbol.toStringTag, {
   enumerable: false,
   writable: true,
   configurable: false,
   value: "process",
 });
+
+export const removeListener = process.removeListener;
+export const removeAllListeners = process.removeAllListeners;
+export const stderr = process.stderr;
+export const stdin = process.stdin;
+export const stdout = process.stdout;
+
+export default process;
+
+//TODO
```

Comment:
Now we require tags for every TODO comment. ref: https://github.com/denoland/deno/pull/9135

Reply:
Done